Library/PagedContainer/PagedContainerViewController.swift

```diff
@@ -0,0 +1,138 @@
+import Combine
+import Foundation
+import UIKit
+
+open class PagedContainerViewController: UIViewController {
+  private weak var activeController: UIViewController? = nil
+  private var subscriptions = Set<AnyCancellable>()
+  private let viewModel = PagedContainerViewModel()
+
+  // TODO: Use the correct page control, per the designs.
+  // This may exist already in SortPagerViewController, or we can write one in SwiftUI.
```

Comment:
Worth calling out - we have another ticket for implementing the correct page control. I leave it up to someone else to determine if we should write a new one in SwiftUI or steal/refactor out the existing one.